.github/workflows/scanner-nvd-update.yaml

```diff
@@ -28,21 +70,12 @@ jobs:
     - name: Set up Cloud SDK
       uses: google-github-actions/setup-gcloud@v2
 
-    - name: Fetch CVEs
-      env:
-        SCANNER_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
-        SCANNER_NVD_URL: https://services.nvd.nist.gov/rest/json/cves/2.0
-      run: |
-        set -eu
-        dir=$(mktemp -d)
-        python3 .github/workflows/scripts/scanner-update-nvd.py "$dir"
-        mkdir nvd-bundle
-        tar -czf nvd-bundle/nvd-data.tar.gz -C "$dir" .
-        gsutil cp -r nvd-bundle "gs://definitions.stackrox.io/v4"
+    - name: Upload production
+      run: gsutil cp nvd.zip "gs://definitions.stackrox.io/v4/nvd/nvd.zip"
 
   send-notification:
     needs:
-    - fetch-cves
+    - upload
```

Comment:
does this still run if `upload` is skipped but `fetch-cves` fails?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds